src/uct/cuda/cuda_ipc/cuda_ipc_ep.c

```diff
@@ -22,19 +22,30 @@
 #define UCT_CUDA_IPC_PUT 0
 #define UCT_CUDA_IPC_GET 1
 
+
 static UCS_CLASS_INIT_FUNC(uct_cuda_ipc_ep_t, const uct_ep_params_t *params)
 {
     uct_cuda_ipc_iface_t *iface = ucs_derived_of(params->iface,
                                                  uct_cuda_ipc_iface_t);
+    const uct_cuda_ipc_iface_addr_t *iface_addr;
 
     UCT_EP_PARAMS_CHECK_DEV_IFACE_ADDRS(params);
     UCS_CLASS_CALL_SUPER_INIT(uct_base_ep_t, &iface->super);
 
+    iface_addr = (const uct_cuda_ipc_iface_addr_t*)params->iface_addr;
+
+    self->keepalive = uct_ep_keepalive_create(iface_addr->pid,
+                                              iface_addr->starttime);
```

Comment:
can we avoid passing starttime in iface address? this increases iface address size?
instead, keepalive_create will read ctime itself